Misc/NEWS.d/next/C API/2024-07-21-17-40-07.gh-issue-121489.SUMFCr.rst

```diff
@@ -0,0 +1 @@
+Export private :c:func:`_PyBytes_Join` again.
```

Comment:
```suggestion
Export private :c:func:`!_PyBytes_Join` again.
```

Sorry for the incomplete suggestion earlier. The function is undocumented so we can't link to it.
But we still can format it as a function.